static/src/javascripts/projects/common/modules/atoms/quiz.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'lib/fastdom-promise';
+
+export default {
```

Comment:
Is there a way we could only export `handleCompletion` as a standalone function?